var/spack/repos/builtin/packages/bazel/package.py

```diff
@@ -90,7 +90,7 @@ class Bazel(Package):
     # Until https://github.com/spack/spack/issues/14058 is fixed, use jdk to build bazel
     # Strict dependency on java@8 as per
     # https://docs.bazel.build/versions/master/install-compile-source.html#bootstrap-unix-prereq
-    depends_on('jdk@1.8.0:1.8.999', type=('build', 'run'))
+    depends_on('java', type=('build', 'run'))
```

Comment:
Is this needed because JDK 8 doesn't work for you, or because JDK doesn't work for you? As you can tell by the comments, this used to be `java@8` but we changed it to `jdk@1.8.0:1.8.999` because the concretizer is broken.

Reply:
trying to stay with openjdk across the board for our installs, not because it doesn't work. As far as a data point, my recent builds have worked with openjdk@11.0.2

Reply:
The docs explicitly state that Java 8 is the only version supported, but I know a lot of people who can't get the build to work at all with Java 8, and it works fine with OpenJDK 11. No idea what's going on there.

Reply:
At some point we should try setting back to java. I discovered that hitting process limits will report an out-of-memory error. Since the bazel build now respects spack's job limits, perhaps this will be less of an issue...